package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "justice-ui-library",
-  "version": "0.11.7",
+  "version": "0.11.8",
```

Comment:
could we bump the version laterin this open [PR](https://github.com/AccelByte/justice-ui-library/pull/111)? both are small fixes, so i think no need to bump the version twice as we also already bump the version today and yet to merge it to AP

Reply:
and yeah i think we can remove #patch from the commit message. wdyt?

Reply:
okay, revert to 0.11.7